examples/process_gain.py

```diff
@@ -105,9 +103,7 @@ def determine_power_from_fit(filename, guessamp, guessph):
 
 input_power = determine_power_from_fit(file1, 5e-2, 0.75)
 input_power.name("Input Power").set_plot_color("r")
-output_power = determine_power_from_fit(
-    file2, 15e-2, 0.75
-)
+output_power = determine_power_from_fit(file2, 15e-2, 0.75)
```

Comment:
most likely same black issue

Reply:
yes, I'm running black here with standard line length --  we want to re-run when copying file to manuscript